src/network/netdev/xfrm.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include "netdev/netdev.h"
+
+typedef struct Xfrm {
+        NetDev meta;
+
+        uint32_t if_id;
+        char	*dev;
```

Comment:
we usually don't align things like this, i.e. no extra inner whitespace (unless there's also a comma)